ui/src/templates/utils/index.ts

```diff
@@ -100,10 +100,12 @@ const getTemplateDetailsFromFileSource = (_source: string): TemplateDetails => {
 
 export const getTemplateNameFromUrl = (
   url: string
-): {name: string; extension: string} => {
-  const fullName = url.split('/').pop()
+): {name: string; extension: string; directory: string} => {
+  const urlSplit = url.split('/')
+  const fullName = urlSplit.pop()
+  const directory = urlSplit.pop()
   const [name, extension] = fullName.split('.')
-  return {name, extension}
+  return {name, extension, directory}
```

Comment:
can you add assertions to the tests here: https://github.com/influxdata/influxdb/blob/master/ui/src/templates/utils/index.test.ts#L51 to make sure directory is what you expect? let me know if you'd like help on this one :)

Reply:
i have added the assertions to the tests!